pkg/quota_management/billing_model.go

```diff
@@ -0,0 +1,10 @@
+package quota_management
+
+type BillingModel struct {
```

Comment:
Another side comment:
With recent work I've been doing, also looking at our code and the code of this PR I think we have quite a lot of Go types overload of the "billing model" concept and also strings named that way.

I think it would be a good idea to have some discussion around this subject through other channels

Reply:
Very good point.